Library/src/main/java/com/shopify/testify/ScreenshotRule.kt

```diff
@@ -92,7 +92,7 @@ typealias ExclusionRectProvider = (rootView: ViewGroup, exclusionRects: MutableS
 @Suppress("unused")
 open class ScreenshotRule<T : Activity> @JvmOverloads constructor(
     private val activityClass: Class<T>,
-    @IdRes private var rootViewId: Int = android.R.id.content,
+    @IdRes var rootViewId: Int = android.R.id.content,
```

Comment:
nit: I think we should turn this into a val if we want to make it public and then provide a method to set it instead of modifying the variable directly.

Reply:
What I wanted was a `private set` but you can't do that on constructor variables.

What I'll probably do is just add a `getRootViewId()` method and make this private again.